File: 118. Pascal's Triangle/118.cs
public class Solution {
    public IList<IList<int>> Generate(int numRows) {
        var re = new List<IList<int>>();
        
        for(int i = 0; i < numRows; i++){
            var tmp = new List<int>();
            tmp.Add(1);
            for(int j = 1; j < i; j++ ){
                tmp.Add(re[i-1][j-1] + re[i-1][j]);
            }
            if(i > 0)
                tmp.Add(1);
            re.Add(tmp);
        }
        
        return re;
    }
}

File: 118. Pascal's Triangle/118.py
# Input: 5
# Output:
# [
#      [1],
#     [1,1],
#    [1,2,1],
#   [1,3,3,1],
#  [1,4,6,4,1]
# ]

class Solution:
    def generate(self, numRows: int) -> List[List[int]]:
        re = []
        for i in range(numRows):
            tmp = [0]*(i+1)
            for j in range(i+1):
                if(j == 0):
                    tmp[0] = 1
                elif(j == i):
                    tmp[j] = 1
                else:
                    print(j)
                    tmp[j] = re[i-1][j-1] + re[i-1][j]
            re.append(tmp)
        return re
        
                